### DISCUSSION TITLE
Allow links function to access path params.

### DISCUSSION THREAD

**Discussion by nicksrandall** [remix-run/remix#5814] [Proposals]
I have a use case where I need to load a stylesheet based on a path param on my page. Seems like we could give the links function access to the params of the current page. 

```js
export const links: LinksFunction = (args) => {
  const { customerId } = args.params;
  return [
    {
      rel: "stylesheet",
      href: `/api/v1/theme/${customerId}`,
    },
  ];
};
```

**Comment by sergiodxa** [comment]
I think this can make sense, path params are known before the loader runs so they don't depend on any async data to be preloaded, this way it doesn't affect `<Link prefetch>`.
